tasks/18_humanize-format/humanize-format.js

```diff
@@ -0,0 +1,17 @@
+export function humanizeFormat(num) {
+  if (num === undefined) return '';
+
+  const firstDigit = num % 10;
+  const secondDigit = (num / 10) % 10;
```

Comment:
These two are more likely to be `lastDigit` and `lastTwoDigits.
And I'm not sure that this `const secondDigit = (num / 10) % 10;` will return what is expected to be returned here. 
I think that this would be solution `const secondDigit = num % 100;` and later:

```
    case 1:
      return secondDigit === 11 ? num + 'th' : num + 'st';
    case 2:
      return secondDigit === 12 ? num + 'th' : num + 'nd';
    case 3:
      return secondDigit === 13 ? num + 'th' : num + 'rd';
```

Reply:
I should have used `Math.trunc( num / 10) % 10` that way it would have worked, but I think your solution is cleaner so I implemented it.